design/core_policies.md

```diff
@@ -0,0 +1,31 @@
+# Node.js core Security Policies
```

Comment:
why is the directory called design ?

Reply:
So, I am not sure this should go in "process" but so far I had not better idea